bin/riak-bucket-exporter.js

```diff
@@ -22,9 +23,16 @@ var count = 0;
 var openWrites = 0;
 var db = require("riak-js").getClient({host: program.host, port: program.port});
 var fs = require('fs');
+var deleteKeys = false;
```

Comment:
how about `var deleteKeys = !!program.delete` then you can save the hassle of reassigning this further down. but the beauty of this is open for discussions ;)